Sources/UrlFormEncoding/UrlFormDecoder.swift

```diff
@@ -163,7 +163,7 @@ public final class UrlFormDecoder: Decoder {
     }
 
     func decodeNil(forKey key: Key) throws -> Bool {
-      return try self.checked(key, ^\.isEmpty)
+      return self.container[key.stringValue].flatMap(self.decoder.unbox).map(^\.isEmpty) ?? true
```

Comment:
We want to allow for `nil`! This lets us specify, in our router, optional things like:

``` swift
Route.iso.pricing
  <¢> get %> lit("pricing") %> queryParams(Pricing?.self) <% end
```